C2GUILauncher/src/InstallHelpers.cs

```diff
@@ -0,0 +1,77 @@
+﻿using C2GUILauncher.JsonModels;
+using Microsoft.Win32;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace C2GUILauncher.src

```

Comment:
remove the src dir here. Visual Studio insists on adding it, but it shouldn't be here